### PR TITLE
updating docker hash for rocm build and test

### PR DESCRIPTION
**Pull request by janeyx99**
Docker hash taken from https://app.circleci.com/pipelines/github/pytorch/pytorch/236259/workflows/e98ed9e4-bbbe-4953-8c35-595841590319/jobs/8762364

### CODE REVIEW

File: src/main/groovy/ossci/caffe2/DockerVersion.groovy [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // This file is manually managed
 package ossci.caffe2
 class DockerVersion {
-  static final String version = "39ff7a31e97a10fc7c0f45583a8d3d6bc348174e";  // WARNING: if you change this to a new version number, you **MUST** also add that version number to the allDeployedVersions list below
+  static final String version = "ec8a95e50f50bca25666a5d3330407aedd1e2f49";  // WARNING: if you change this to a new version number, you **MUST** also add that version number to the allDeployedVersions list below
```
Comment (jeffdaily) [inline src/main/groovy/ossci/caffe2/DockerVersion.groovy:4]:
```suggestion
  static final String version = "9e980b21bb85da4062fe89410f9d6af5c850fbca";  // WARNING: if you change this to a new version number, you **MUST** also add that version number to the allDeployedVersions list below
```

File: src/main/groovy/ossci/pytorch/DockerVersion.groovy [hunk 2]
```diff
@@ -7,7 +7,7 @@ package ossci.pytorch
 class DockerVersion {
   // WARNING: if you change this to a new version number,
   // you **MUST** also add that version number to the allDeployedVersions list below
-  static final String version = "39ff7a31e97a10fc7c0f45583a8d3d6bc348174e";
+  static final String version = "ec8a95e50f50bca25666a5d3330407aedd1e2f49";
```
Comment (jeffdaily) [inline src/main/groovy/ossci/pytorch/DockerVersion.groovy:10]:
```suggestion
  static final String version = "9e980b21bb85da4062fe89410f9d6af5c850fbca";
```

**Comment by jeffdaily** [comment]
Latest commit hash from your associated pytorch PR appears to have changed.

From this log:
https://app.circleci.com/pipelines/github/pytorch/pytorch/236796/workflows/9fe48e21-754a-48cc-9cac-3a9a2193eef8/jobs/8793383

Looks like new hash is: `9e980b21bb85da4062fe89410f9d6af5c850fbca`

**Comment by janeyx99** [comment]
> Latest commit hash from your associated pytorch PR appears to have changed.
> 
> From this log:
> https://app.circleci.com/pipelines/github/pytorch/pytorch/236796/workflows/9fe48e21-754a-48cc-9cac-3a9a2193eef8/jobs/8793383
> 
> Looks like new hash is: `9e980b21bb85da4062fe89410f9d6af5c850fbca`

You're right, thank you for the catch!

**Comment by jeffdaily** [comment]
LGTM.  Shall we merge?  I have merge authority.

**Comment by janeyx99** [comment]
Yes please thank you!

**Pull request merged**